backend/user/views.py

```diff
@@ -43,6 +43,10 @@ def post(self, request):
         except Exception as e:
             raise APIException(e)
 
+class DeleteUser(APIView):
+    def delete(self, request: HttpRequest):
```

Comment:
This does not work. I have relayed it to a separate issue.

Reply:
You're right that this needs more authentication. I've created a new issue to keep track: # 66.